packages/@react-aria/tree/src/useTreeGridListItem.ts

```diff
@@ -36,6 +43,27 @@ export interface TreeGridListItemAria extends GridListItemAria {
  * @param ref - The ref attached to the row element.
  */
 export function useTreeGridListItem<T>(props: AriaTreeGridListItemOptions, state: TreeState<T>, ref: RefObject<FocusableElement>): TreeGridListItemAria {
-   // TODO: should it return a state specifically for isExpanded? Or is aria attribute sufficient?
-  return useGridListItem(props, state, ref);
+  let {node} = props;
+  let gridListAria = useGridListItem(props, state, ref);
+  let isExpanded = gridListAria.rowProps['aria-expanded'] === true;
+  let stringFormatter = useLocalizedStringFormatter(intlMessages, 'react-aria-components');
+
+  let expandButtonProps = {
+    onPress: () => {
+      if (!gridListAria.isDisabled) {
+        state.toggleKey(node.key);
+        state.selectionManager.setFocused(true);
+        state.selectionManager.setFocusedKey(node.key);
+      }
+    },
+    'aria-label': isExpanded ? stringFormatter.format('collapse') : stringFormatter.format('expand'),
+    tabIndex: isAndroid() ? -1 : null,
+    'data-react-aria-prevent-focus': true
```

Comment:
Oh, we already found another use case for this huh?? 😄

Reply:
yup, will help simplify the table's expandable row chevron when we add that to RAC as well. I can probably combine it with `preventFocusOnPress` to make the chevron in RAC tree completely unfocusable outside of mobile screen readers but will dig into that more after release haha